### PR TITLE
Update CVE-2020-13668.json

### PR DESCRIPTION
**Pull request by mlhess**

### CODE REVIEW

File: 2020/13xxx/CVE-2020-13668.json [hunk 1]
```diff
@@ -1,18 +1,109 @@
 {
```
Comment (cve-team) [inline 2020/13xxx/CVE-2020-13668.json:1]: 'data_type' is a required property

Comment (cve-team) [inline 2020/13xxx/CVE-2020-13668.json:1]: 'data_format' is a required property

Comment (cve-team) [inline 2020/13xxx/CVE-2020-13668.json:1]: 'data_version' is a required property

Comment (cve-team) [inline 2020/13xxx/CVE-2020-13668.json:1]: 'CVE_data_meta' is a required property

Comment (cve-team) [inline 2020/13xxx/CVE-2020-13668.json:1]: 'affects' is a required property

Comment (cve-team) [inline 2020/13xxx/CVE-2020-13668.json:1]: 'problemtype' is a required property

Comment (cve-team) [inline 2020/13xxx/CVE-2020-13668.json:1]: 'references' is a required property

Comment (cve-team) [inline 2020/13xxx/CVE-2020-13668.json:1]: 'description' is a required property

**Review by cve-team — CHANGES_REQUESTED** [review]
Auto validation errors:

File: 2020/13xxx/CVE-2020-13668.json [hunk 2]
```diff
@@ -1,18 +1,110 @@
...
```
Comment (cve-team) [inline 2020/13xxx/CVE-2020-13668.json:1]: 'data_type' is a required property

Comment (cve-team) [inline 2020/13xxx/CVE-2020-13668.json:1]: 'data_format' is a required property

Comment (cve-team) [inline 2020/13xxx/CVE-2020-13668.json:1]: 'data_version' is a required property

Comment (cve-team) [inline 2020/13xxx/CVE-2020-13668.json:1]: 'CVE_data_meta' is a required property

Comment (cve-team) [inline 2020/13xxx/CVE-2020-13668.json:1]: 'affects' is a required property

Comment (cve-team) [inline 2020/13xxx/CVE-2020-13668.json:1]: 'problemtype' is a required property

Comment (cve-team) [inline 2020/13xxx/CVE-2020-13668.json:1]: 'references' is a required property

Comment (cve-team) [inline 2020/13xxx/CVE-2020-13668.json:1]: 'description' is a required property

**Review by cve-team — CHANGES_REQUESTED** [review]
Auto validation errors:

**Comment by anthnysingleton** [comment]
Hey @mlhess,

I may have given you the wrong link. Use this one for creating JSON V4 files.
https://vulnogram.github.io/#editor

The other link I gave you was for the v5 editor, which we cannot accept through GitHub at this time
https://vulnogram.github.io/cve5/#json

Thanks,
Anthony

**Comment by mlhess** [comment]
I think my last submission was from the older version, can you rerun the bot on it?

**Comment by anthnysingleton** [comment]
Correct the commits you submitted to resolve the validation errors worked and your PR was automatically approved and merged. 

If you are part of the CVE slack communication we can communicate in real time to cover any additional questions you may have.

Thanks,
Anthony

**Comment by mlhess** [comment]
How can I join the slack?

**Comment by anthnysingleton** [comment]
I will get an invite sent to you shortly.

**Pull request merged**